packages/patternfly-react/src/components/SessionTimeout/helpers.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export class Session extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      now: this.props.now(),
+      expiresAt: this.props.expiresAt
+    };
+  }
+
+  componentWillMount() {
+    this._interval = setInterval(
```

Comment:
I am thinking a setTimeout would be better here instead of an interval. We can call a callback once it expires instead of rendering every second. This component seems like it is meant for the top level, and will have a huge performance issue with rendering this often.